packages/tasit-sdk/src/testHelpers/helpers.js

```diff
@@ -31,69 +31,84 @@ const gasParams = {
 };
 
 const setupWallets = () => {
-  const owner = createFromPrivateKey(ownerPrivKey);
-  const seller = createFromPrivateKey(sellerPrivKey);
-  const buyer = createFromPrivateKey(buyerPrivKey);
-  const ephemeral = Account.create();
+  const ownerWallet = createFromPrivateKey(ownerPrivKey);
+  const sellerWallet = createFromPrivateKey(sellerPrivKey);
+  const buyerWallet = createFromPrivateKey(buyerPrivKey);
+  const ephemeralWallet = Account.create();
 
-  return { owner, seller, buyer, ephemeral };
+  return { ownerWallet, sellerWallet, buyerWallet, ephemeralWallet };
 };
 
-const setupContracts = async owner => {
+const setupContracts = async ownerWallet => {
   // Note: It would be cooler to use NFT here if
   // Decentraland Land contract followed ERC721 exactly
-  const land = new Contract(landAddress, landABI, owner);
-  const landProxy = new Contract(landProxyAddress, landProxyABI, owner);
-  const estate = new Contract(estateAddress, estateABI, owner);
+  const landContract = new Contract(landAddress, landABI, ownerWallet);
+  const landProxyContract = new Contract(
+    landProxyAddress,
+    landProxyABI,
```

Comment:
It's definitely the ABI of the land proxy and not the land contract?

Reply:
I see elsewhere that we kept it as landABI

Reply:
There are both ABIs, after their initializations, the contract that is been exposing to test suite is the proxy one but with the land ABI.